apiserver/facades/controller/instancepoller/mock_test.go

```diff
@@ -423,47 +411,13 @@ func (m *mockMachine) Status() (status.StatusInfo, error) {
 	return m.status, m.NextErr()
 }
 
-type mockLinkLayerDeviceAddress struct {
-	configMethod     network.AddressConfigMethod
-	subnetCIDR       string
-	dnsServers       []string
-	dnsSearchDomains []string
-	gatewayAddress   string
-	isDefaultGateway bool
-	value            string
-}
-
-func (m mockLinkLayerDeviceAddress) ConfigMethod() network.AddressConfigMethod { return m.configMethod }
-func (m mockLinkLayerDeviceAddress) SubnetCIDR() string                        { return m.subnetCIDR }
-func (m mockLinkLayerDeviceAddress) DNSServers() []string                      { return m.dnsServers }
-func (m mockLinkLayerDeviceAddress) DNSSearchDomains() []string                { return m.dnsSearchDomains }
-func (m mockLinkLayerDeviceAddress) GatewayAddress() string                    { return m.gatewayAddress }
-func (m mockLinkLayerDeviceAddress) IsDefaultGateway() bool                    { return m.isDefaultGateway }
-func (m mockLinkLayerDeviceAddress) Value() string                             { return m.value }
-
-type mockLinkLayerDevice struct {
-	name             string
-	mtu              uint
-	devType          network.LinkLayerDeviceType
-	isLoopbackDevice bool
-	macAddress       string
-	isAutoStart      bool
-	isUp             bool
-	parentName       string
-
-	addresses []instancepoller.StateLinkLayerDeviceAddress
-}
-
-func (m mockLinkLayerDevice) Name() string                      { return m.name }
-func (m mockLinkLayerDevice) MTU() uint                         { return m.mtu }
-func (m mockLinkLayerDevice) Type() network.LinkLayerDeviceType { return m.devType }
-func (m mockLinkLayerDevice) IsLoopbackDevice() bool            { return m.isLoopbackDevice }
-func (m mockLinkLayerDevice) MACAddress() string                { return m.macAddress }
-func (m mockLinkLayerDevice) IsAutoStart() bool                 { return m.isAutoStart }
-func (m mockLinkLayerDevice) IsUp() bool                        { return m.isUp }
-func (m mockLinkLayerDevice) ParentName() string                { return m.parentName }
-func (m mockLinkLayerDevice) Addresses() ([]instancepoller.StateLinkLayerDeviceAddress, error) {
-	return m.addresses, nil
+// AssertAliveOp implements StateMachine.
+func (m *mockMachine) AssertAliveOp() txn.Op {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	m.MethodCall(m, "Status")
+	return txn.Op{C: "machine-alive"}
```

Comment:
Being a little creative with ops, so we can interrogate which were generated and in which order.